lib/services/schema-object-factory.ts

```diff
@@ -161,7 +161,7 @@ export class SchemaObjectFactory {
 
       const schemaCombinators = ['oneOf', 'anyOf', 'allOf'];
       if (schemaCombinators.some((key) => key in property)) {
-        delete (property as SchemaObjectMetadata).type;
+        (property as SchemaObjectMetadata).type = 'object'
```

Comment:
Didn't label as a breaking change as this is adding not removing properties. Based on my testing OpenAPI validators were happy with this addition.